From 3f41521d4b9697634d3f574ee5fd3a22f217b25d Mon Sep 17 00:00:00 2001
From: Siddhesh Poyarekar <siddhesh@redhat.com>
Date: Mon, 15 Sep 2014 10:08:38 +0530
Subject: [PATCH] Add correct variable names for _POSIX_IPV6 and
 _POSIX_RAW_SOCKETS

getconf only recognizes IPV6 and RAW_SOCKETS, when the standard
requires it to recognize the actual configuration variable name[1].  I
have not removed the earlier names for compatibility.

	* posix/getconf.c (vars): Add _POSIX_IPV6 and
	_POSIX_RAW_SOCKETS.

[1] http://pubs.opengroup.org/onlinepubs/007904875/functions/sysconf.html
---
 posix/getconf.c | 7 +++++++
 2 files changed, 12 insertions(+)

diff --git a/posix/getconf.c b/posix/getconf.c
index db6acc4..c2c0d3f 100644
--- a/posix/getconf.c
+++ b/posix/getconf.c
@@ -1004,6 +1004,13 @@ static const struct conf vars[] =
     { "RAW_SOCKETS", _SC_RAW_SOCKETS, SYSCONF },
 #endif
 
+#ifdef _SC_IPV6
+    { "_POSIX_IPV6", _SC_IPV6, SYSCONF },
+#endif
+#ifdef _SC_RAW_SOCKETS
+    { "_POSIX_RAW_SOCKETS", _SC_RAW_SOCKETS, SYSCONF },
+#endif
+
     { NULL, 0, SYSCONF }
   };
 
-- 
2.1.2

